### DISCUSSION TITLE
Update crashes Home Assistant machine

### DISCUSSION THREAD

**Discussion by jwildman16** [babybuddy/babybuddy#581] [Q&A] [answered]
I've got Baby Buddy installed as a Home Assistant add-on. IIRC, that's not officially supported, so I didn't want to create an issue here. However, when attempting to upgrade to the latest version (I think 1.13.2.1 to 1.13.2.4, but I'm going based on memory bec I can't pull it up right now), Home Assistant crashes entirely. It seems it may be a resource issue because I can start an SSH session with the rPi3 running Home Assistant OS, but can't actually do anything with it. It just hangs. This has happened at least twice now and I was able to resolve it last time (but not complete the upgrade) by pulling power on the rPi and letting it restart (never like doing that). 

Has anyone else experienced this or have a resolution? Once I'm up and running again, I'll probably just uninstall and then reinstall using the latest version.

**Comment by cdubz** [comment, marked as answer]
I think this is related to https://github.com/OttPeterR/addon-babybuddy? You could open an issue there or we can try to summon @OttPeterR here 😃

**Reply by jwildman16** [reply]
Thank you. Makes sense that there's a separate repo for that. I ended up letting it go all night and the update was eventually successful and HA came back on its own.
